controller_names:
  - joint_trajectory_controller

joint_trajectory_controller:
  type: "joint_trajectory_controller/JointTrajectoryController"
  action_ns: "follow_joint_trajectory"
  default: True
  joints:
    - joint_1
    - joint_2
    - joint_3
    - joint_4
    - joint_5
    - joint_6
  gains: # Optional, but often required if the joint limits are missing (see WARN)
    joint_1: {p: 100.0, i: 0.01, d: 1.0}
    joint_2: {p: 100.0, i: 0.01, d: 1.0}
    joint_3: {p: 100.0, i: 0.01, d: 1.0}
    joint_4: {p: 100.0, i: 0.01, d: 1.0}
    joint_5: {p: 100.0, i: 0.01, d: 1.0}
    joint_6: {p: 100.0, i: 0.01, d: 1.0}
